torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3761,9 +3787,36 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                                 dtypes=[torch.int],
                                 active_if=IS_WINDOWS),
                    )),
+    UnaryUfuncInfo('conj_physical',
+                   ref=np.conj,
+                   dtypes=all_types_and_complex_and(torch.bool,
+                                                    torch.bfloat16, torch.half),
+                   dtypesIfCPU=None,
```

Comment:
Note: not for this PR, but in the future we should make the defaults for these entries in the UnaryUfuncInfos to be None. The fact that they're non-None is an artifact of how the OpInfos were first constructed, and doesn't fit our current work that well

cc @kshitij12345 @imaginary-person